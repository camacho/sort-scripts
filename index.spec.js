const sortScripts = require('./index.js');

describe('sortScripts', () => {
  it('sorts based on prefix and suffix', () => {
    const scripts = {
      test: 'jest',
      pretest: 'check-engines',
      postmerge: 'lint-staged',
      docs: 'markdown-magic',
      postdocs: 'echo "\'.md\' files updated"',
    };

    expect(sortScripts(scripts)).toMatchSnapshot();
  });
});
