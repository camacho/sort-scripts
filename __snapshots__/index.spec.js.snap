// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sortScripts sorts based on prefix and suffix 1`] = `
Array [
  Array [
    "docs",
    "markdown-magic",
  ],
  Array [
    "postdocs",
    "echo \\"'.md' files updated\\"",
  ],
  Array [
    "postmerge",
    "lint-staged",
  ],
  Array [
    "pretest",
    "check-engines",
  ],
  Array [
    "test",
    "jest",
  ],
]
`;
